### DISCUSSION TITLE
bluesky-docenienie-zmiany-domena-jako-identyfikator/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#128] [General]
# bluesky-docenienie-zmiany-domena-jako-identyfikator/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/bluesky-docenienie-zmiany-domena-jako-identyfikator/?utm_medium=rss&utm_source=rss&utm_campaign=rss

**Comment by spexpl** [comment]
1) Czy nick musi być w ogóle z jakaś domeną? Czy nie może być po prostu @idarek ??
2) A jeśli potrzeba jednak jakaś domenę, to w jaki sposób można zrobić konto "serwisowi"?? czy przejdzie po prostu @wieckiewicz.org, czy trzeba kombinować z @official.wieckiewicz.org ??

**Reply by idarek** [reply]
1. Nie, nie może. Bluesky, mimo że wygląda jak Twitter kiedyś, to nie to samo. Podobnie jak w MAstodon, jeżeli chcesz zostać odnaleziony, wówczas potrzebny jest jeszcze serwer na którym użytkownik jest zarejestrowany.
2. Możesz używać ich, wówczas twój użytkownik jest w domenie @użytkownik.bsky.social. Można zrobić konto główne a pożniej użytkownikom w domenie. Tak jak opisują oni w daleszej części ich tutorialu (**How can I set and manage multiple subdomain handles?**). Ja mógłbym wybrać po prostu moją główną domenę, jako identyfikator, ale jako że mam to rozplanowane inaczej (domena główna nie kieruje na żadną stronę główną), wybrałem `@dariusz.wieckiewicz.org`.

**Reply by spexpl** [reply]
ad.2 Czyli to coś ala CNAME domen? 

PS. Żona robiła korektę? Bo trochę -łam

**Reply by idarek** [reply]
Tak :)
Ale TXT rekord to tylko dodatek, który nic nie zmienia, a używany jest do różnych celów, np. antyspamowych czy weryfikacji.
